CONTRIBUTING.rst

```diff
@@ -128,11 +128,11 @@ Writing your first Pull Request
 Preparing the ground
 ^^^^^^^^^^^^^^^^^^^^
 
-If not already done, you’ll want to create an issue first, and discuss
+If not already done, first create an issue, and discuss
 the changes with the project’s maintainers.
 
-Please refer to the previous section *How do I submit a (good)
-enhancement proposal?* for more information.
+Please refer to the previous section `*How do I submit a (good)
+enhancement proposal?*<https://skrub-data.org/stable/CONTRIBUTING.html#how-do-i-submit-a-good-enhancement-proposal>`__ for more information.
```

Comment:
Not sure the formatting is right, plus the link changed because the section name changed.